src/components/StatusUpdater/index.tsx

```diff
@@ -8,24 +8,40 @@ import { SplitButton } from '../SplitButton';
 import { useFormLogic, UpdateStatusForm } from './UpdateStatusForm';
 
 const options = [
-  { id: ResponderStatus.Unavailable, text: 'Not Available' },
   { id: ResponderStatus.Standby, text: 'Stand By' },
   { id: ResponderStatus.SignedIn, text: 'Sign In' },
   { id: ResponderStatus.SignedOut, text: 'Sign Out' },
 ]
 const optionTexts = options.reduce((accum, cur) => ({ ...accum, [cur.id]: cur.text }), {} as Record<string, string>);
 
+function isFuture(startTime: number) {
+  return (startTime - 60 * 60 * 1000) > new Date().getTime();
+};
+
 function getRecommendedAction(current: ResponderStatus|undefined, startTime: number): ResponderStatus {
-  const now = new Date().getTime();
   if (current === ResponderStatus.SignedIn) {
     return ResponderStatus.SignedOut;
   }
-  if (startTime - 60 * 60 * 1000 > now) {
-    return ResponderStatus.Standby;
+  if (isFuture(startTime)) {
+    return current === ResponderStatus.Standby ? ResponderStatus.SignedOut : ResponderStatus.Standby;
   }
   return ResponderStatus.SignedIn;
 }
 
+function getCurrentOptions(current: ResponderStatus|undefined, startTime: number) {
+  if (current === undefined) { return options; }
+  if (isFuture(startTime)) {
+    if (current === ResponderStatus.Standby) {
+      return options.filter(option => option.id === ResponderStatus.SignedOut);
+    }
+    return options.filter(option => option.id === ResponderStatus.Standby);
+  }
+  if (current === undefined || current === ResponderStatus.Unavailable) {

```

Comment:
Hmm... I didn't realize that guard clause was at the top, that actually needs to be below the future use case and should never return all options. I removed the unavailable status as an option here, but it's still a responder status and I was seeing records come through in that state when the user hasn't interacted with that mission yet. I think both cases need to be handled, here, but I need to remove the guard clause above. They have to be handled as a separate use case because sign out shouldn't be an option to a mission that you haven't signed into.

Reply:
Ahh ok I'm on board now. Agreed.